src/openms/source/APPLICATIONS/TOPPBase.cpp

```diff
@@ -169,6 +169,7 @@ namespace OpenMS
     registerStringOption_("write_ctd", "<out_dir>", "", "Writes the common tool description file(s) (Toolname(s).ctd) to <out_dir>", false, true);
     registerStringOption_("write_wsdl", "<file>", "", "Writes the default WSDL file", false, true);
     registerFlag_("no_progress", "Disables progress logging to command line", true);
+    registerFlag_("force", "Overwrite tool specific checks.", true);
```

Comment:
maybe we could add some documentation for each tool what the force option is doing in this context and here we should write that it is context dependent?